### PR TITLE
Issue #KN-617 feat: Video Player angular version update from 12 to 13

### PR DESCRIPTION
**Pull request by santoshilimi**
Implementation Video: https://drive.google.com/file/d/1I6AwHqGI6Qzv4q2TIs-Dpr54ovOHfQ0O/view?usp=sharing

<table>
<td>
  <a href="https://project-sunbird.atlassian.net/browse/KN-617" title="KN-617" target="_blank">RFCKN-617</a>  Video Player angular version update from 12 to 14 with IVY
  </td></table>
  <br />
 

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: projects/video-player-wc/src/test.ts [hunk 1]
```diff
@@ -17,7 +17,9 @@ declare const require: {
 // First, initialize the Angular testing environment.
 getTestBed().initTestEnvironment(
   BrowserDynamicTestingModule,
-  platformBrowserDynamicTesting()
+  platformBrowserDynamicTesting(), {
+    teardown: { destroyAfterEach: false }
```
Comment (HarishGangula) [inline projects/video-player-wc/src/test.ts:21]: Why this is false? it will increase the test case running time

File: src/test.ts [hunk 2]
```diff
@@ -12,7 +12,9 @@ declare const require: any;
 // First, initialize the Angular testing environment.
 getTestBed().initTestEnvironment(
   BrowserDynamicTestingModule,
-  platformBrowserDynamicTesting()
+  platformBrowserDynamicTesting(), {
+    teardown: { destroyAfterEach: false }
```
Comment (HarishGangula) [inline src/test.ts:16]: Same applies here

**Review by HarishGangula — CHANGES_REQUESTED** [review]

**Comment by HarishGangula** [comment]
README file also not updated

File: projects/video-player-wc/src/test.ts (hunk 1 above)
Comment (santoshilimi) [reply projects/video-player-wc/src/test.ts:21]: Angular migration itself added this , manually i have not changed.

File: src/test.ts (hunk 2 above)
Comment (santoshilimi) [reply src/test.ts:16]: Angular migration itself added this , manually i have not changed.

**Review by HarishGangula — APPROVED** [review]

**Pull request merged**